README.md

```diff
@@ -12,18 +12,18 @@ Robolectric supports running unit tests for *16* different versions of Android,
 Here's an example of a simple test written using Robolectric:
 
 ```java
-@RunWith(AndroidJUnit4.class)
+@RunWith(RobolectricTestRunner.class)
```

Comment:
@seadowg What about keeping using `AndroidJUnit4`?

Reply:
Yeah I'd also recommend using AndroidJUnit4 by default as well.

Reply:
I went with `RobolectricTestRunner` to keep it consistent with http://robolectric.org/. I had switched away from using `AndroidJUnit4` on the website because using that runner requires also having `androidx.test.ext:junit` as a dependency. 

Personally, I like the idea of having Robolectric docs focus on Robolectric in isolation and then have extra docs that describe using it as part of AndroidX Test (like http://robolectric.org/androidx_test/ and Android official docs).

Reply:
@hoisie I agree with @seadowg thought. Maybe we can add new commits to add special examples with `AndroidJUnit4`.

Reply:
Any update on the thinking here?

Reply:
Friendly ping @hoisie .

Reply:
@seadowg What about keeping `AndroidJUnit4` and only changing with `try-catch-with-resource` of `ActivityController`? It might be easier to merge this PR.

Reply:
@utzcoz it looks like you've already approved the PR. Is there any disagreement with the current changes?

Reply:
Yep. I approved it before, and I think this PR is fine to me. But @hoisie has different opinion about switching `AndroidJUnit4` to `RobolectricTestRunner`. I want to know whether this PR only keep changes for `try-catch-resource` modifications and push the merging progress of this PR. And you can create a new PR for discussion.

Reply:
> But @hoisie has different opinion about switching AndroidJUnit4 to RobolectricTestRunner.

Is this coming from a discussion outside this thread? @hoisie still hasn't replied to my initial [comment](https://github.com/robolectric/robolectric/pull/7475#discussion_r934294141).

Reply:
> Yeah I'd also recommend using AndroidJUnit4 by default as well.

@seadowg This one.

Reply:
Sorry for the delay in replying @utzcoz! I'm happy to wait for a reply from @hoisie to my follow-up on that if we're not happy to merge on the single approval:

> I went with RobolectricTestRunner to keep it consistent with http://robolectric.org/. I had switched away from using AndroidJUnit4 on the website because using that runner requires also having androidx.test.ext:junit as a dependency.

I think we'll end up repeating these points in a new PR otherwise, and I don't think the change to `try-with-resources` is high priority enough to need to get through now seeing as we've already made the change to the website.

Reply:
Any update on this? If it's still preferred that we use `AndroidJUnit4` as the runner in examples, then I'd suggest we also add `androidx.test.ext:junit` as a dependency in the "Starting a New Project" section. Happy to go with that if it'd make more sense to everyone!